src/analysis/relaxedFBA/relaxFBA_cappedL1.m

```diff
@@ -50,72 +50,71 @@
 
 [m,n] = size(model.S); %Check inputs
 
-
-if isfield(model,'SIntRxnBool')
-    intRxnBool = model.SIntRxnBool;
-    exRxnBool = ~intRxnBool;
-else
-    model_Ex = findSExRxnInd(model);
-    intRxnBool = model_Ex.SIntRxnBool;
-    exRxnBool = ~intRxnBool;
-end
-
-relaxOption.maxUB = max(max(model.ub),-min(model.lb));
-relaxOption.minLB = min(-max(model.ub),min(model.lb));
+param.maxUB = max(max(model.ub),-min(model.lb));
+param.minLB = min(-max(model.ub),min(model.lb));
 
 stop = false;
 solution.stat = 1;
 
-if exist('relaxOption','var')
-    if isfield(relaxOption,'excludedReactions') == 0
-        relaxOption.excludedReactions = false(n,1);
+
```

Comment:
Just a comment:
This is essentially an input Parser.
If it would be implemented using an input parser (and varargin), you could use both a struct or parameter value pairs and simultaneously check the values for validity.